optimism/src/mips/interpreter.rs

```diff
@@ -55,9 +59,9 @@ pub enum RTypeInstruction {
     Nor,                          // nor
     SetLessThan,                  // slt
     SetLessThanUnsigned,          // sltu
-    MultiplyToRegister,           // mul
-    CountLeadingOnes,             // clo
-    CountLeadingZeros,            // clz
+    MultiplyToRegister,           // mul - FIXME: pseudo?
+    CountLeadingOnes,             // clo - FIXME: pseudo?
+    CountLeadingZeros,            // clz - FIXME: pseudo?
```

Comment:
I don't quite get the "pseudo" comment here. All these instructions are documented in the official instruction set description from [here](https://mips.com/products/architectures/mips32-2/)

Reply:
Removed